extensions/ql-vscode/test/vscode-tests/no-workspace/query-history/store/query-history-store.test.ts

```diff
@@ -277,28 +253,25 @@ describe("write and read", () => {
   function createMockQueryWithResults(
     queryPath: string,
     didRunSuccessfully = true,
-    hasInterpretedResults = true,
     dbPath = "/a/b/c",
-    includeSpies = true,
   ): QueryWithResults {
     // pretend that the results path exists
     const resultsPath = join(queryPath, "results.bqrs");
     mkdirpSync(queryPath);
     writeFileSync(resultsPath, "", "utf8");
 
-    const query = new QueryInProgress(
+    const queryEvalInfo = new QueryEvaluationInfo(
```

Comment:
In here, we were only accessing the `queryEvalInfo` field of the `QueryInProgress`, but the `QueryInProgress` was part of the legacy query server. This should be equivalent, since [the `QueryInProgress` only created the `QueryEvaluationInfo`](https://github.com/github/vscode-codeql/blob/d28cc6e3f234da0d71b8c6e5d6e22f974fcf54b9/extensions/ql-vscode/src/query-server/legacy/run-queries.ts#L167-L193) and did not have any other purpose.